src/SIM/SIMbase.h

```diff
@@ -126,6 +126,10 @@ class SIMbase : public SIMadmin, public SIMdependency
   //! \brief Returns a pointer to the problem-specific data object.
   const IntegrandBase* getProblem() const { return myProblem; }
 
+  //! \brief Returns interface checker type for model.
+  virtual ASM::InterfaceChecker* getInterfaceChecker(size_t pIdx) const
+  { return nullptr; }
```

Comment:
unused argument, make it anonymous

Reply:
overflows 80 chars, which is very meh. all arguments are anon.